README.md

```diff
@@ -39,10 +39,12 @@ There are a several reasons to use TPI instead of other related solutions (custo
    TPI is a CLI tool, not a running service. It requires no additional orchestrating machine (control plane/head nodes) to schedule/recover/terminate instances. Instead, TPI runs (spot) instances via cloud-native scaling groups[^scalers], taking care of recovery and termination automatically on the cloud provider's side. This design reduces management overhead & infrastructure costs. You can close your laptop while cloud tasks are running -- auto-recovery happens even if you are offline.
 2. **Unified tool for data science and software development teams**:
    TPI provides consistent tooling for both data scientists and DevOps engineers, improving cross-team collaboration. This simplifies compute management to a single config file, and reduces time to deliver ML models into production.
+3. **Reproducible, codified environments**:
+   Store hardware requirements in a single configuration file alongside the rest of your ML pipeline code.
```

Comment:
When we number the list it seems like we mean that it is an exhaustive list. Is it? Otherwise maybe bullets?

But also it somewhat overlaps with the previous bullet list. Maybe they can be combined?